docs_src/design/ucr/Extending-Device-Data.md

```diff
@@ -0,0 +1,78 @@
+## Extending Device Data
+This UCR describes the Use Case for Extending of Device Data by Application Services for a given south-bound Device.
+
+### Submitters
+- Tom Brennan (Eaton)
+
+## Change Log
+- [pending](https://github.com/edgexfoundry/edgex-docs/pull/845) (2022-09-12)
+
+
+### Market Segments
+Any that deploy EdgeX systems with analytics, utility, or north-bound microservices that add new Device Resources
+that are extensions of the original south-bound or service-based Device data.
+
+### Motivation
+We find a consistent need as we design microservices for our industrial products:
+The new analytics, utility, and north-bound microservices almost always need to add Device Resources to manage their configuration, transforms, and status reporting. These Resources are usually needed on a per-Device basis (rather than just overall service configuration or status), which can be seen as extending (adding to) the data of the original south-bound devices.
+
+Adding configuration and status via Resources that **extend** the original south-bound Device
+make this configuration and status data easily accessible and translatable to other Application Services and to the UI via REST;
+we think that this general solution is better than disparate solutions which add custom APIs in each Application Service to Get and Set this data.
+
+What is needed is a common means of showing the relationship between these added Resources, their owning service,
+and the original south-bound Device Resources; that is, to indicate that these Resources "extend" the original Device data.
+
+It is desirable that the means of conveying this information become standardized for those EdgeX microservices
+which provide and use it, hence proposing here that there be a common EdgeX way defined to do this.
+
+### Target Users
```

Comment:
a nit, but I thought we decided to get rid of this section in UCR?

Reply:
Farshid asked if we thought it was useful, but I don't think we reached a firm decision on removing it.
It is still in the template for UCRs.